arm-recoveryservicesbackup/2016-12-01/swagger/backupManagement.json

```diff
@@ -18,35 +18,6 @@
     "application/json"
   ],
   "paths": {
-    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations": {
```

Comment:
Our SDK is currently in preview so we will be upgrading minor version for it. CLI is not yet released all it will be including these changes.